robyn/__init__.py

```diff
@@ -99,6 +99,11 @@ def add_route(
             }
             route_type = http_methods[route_type]
 
+        routes = self.router.get_routes()
+
+        for route in routes:
```

Comment:
You don't need to do the for loop here, this function runs for every route separately and has the endpoint and method as input parameters.

Reply:
Sorry, I misunderstand when you said to write the logs in `add_route` method. I tried this without using a for loop, but it shows one route when running the `base_routes.py` file. What I'm trying to do is that it shows all the routes in the list of routes.
```
if len(routes) > 2:
            logger.info(f"Logging endpoint: method={routes[0][0]}, route={routes[0][1]}")
```
But it just shows the logging of the one route when a I run the `base_routes.py` file
![image](https://github.com/sparckles/robyn/assets/70811425/886dbc97-a48d-438e-b4d8-196bbaa5496d)

Reply:
My bad. Sorry, yes I misunderstand.

```
logger.info(f"Logging endpoint: method={route_type}, route={endpoint}")

```
![image](https://github.com/sparckles/robyn/assets/70811425/f6b72677-af75-44cb-b741-6a66c82037b2)